### ISSUE TITLE
Subpanel Save when visible in EditView does not hide form after save complete

### ISSUE THREAD

**Issue by khan01** [salesagility/SuiteCRM#2460]
If we enable Subpanels in EditView of Leads module, creating a subpanel record does not hide the create form although the record is created. 

#### Issue

When Subpanels are visible in Edit View for a module. While creating a record in subpanel, it does the necessary creation, but SubPanelTiles.js breaks in hiding the create form. 

Expected Behavior

While in the Edit View, the subpanel create form should have been replaced by the records added the subpanel. 
#### Actual Behavior

Create form does not hide for subpanel. 
#### Steps to Reproduce

1. Create a custom view.edit.php file for Leads
2.  Put following code in file for enabling subpanels in Edit View.

```
class LeadsViewEdit extends ViewEdit
{
    public function __construct()
    {
        parent::__construct();
        $this->useForSubpanel = true;
        $this->useModuleQuickCreateTemplate = true;
        $this->options['show_subpanels'] = true;
    }

}
```
1. Now go to Edit View of  any Lead record and save a Note or Attachment form subpanels visible in Edit View. 
2. After clicking save button, it will show error in console, while the actual record has been created, the form is still shown and open. 
#### Your Environment

- SuiteCRM Version used: 7.7.4
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):  Version 54.0.2840.59 m
- Environment name and version (e.g. MySQL, PHP 7): PHP 5.5.12
- Operating System and version (e.g Ubuntu 16.04):  Windows 7(64 bit)

**Comment by shogunpol** [comment]
@khan01 , please provide exact path and name of the created file. And is this only one place when you add the code?

**Comment by khan01** [comment]
@shogunpol Path for Custom File is 

> custom\modules\Leads\views\view.edit.php

I had already submitted content of this file above ^

**Comment by shogunpol** [comment]
The subpanels are used only for detailView, not editView. Due to customisation which is not related to intend functionality of application, i want ask you to close this issue.
Thanks for your contribution.

**Comment by khan01** [comment]
@shogunpol  Yes, i understand that they are for detail view. The Option to show them in edit view is from Core SugarCRM code. Yet this great feature reduces the user navigation to different screens in order to modify Parent and its related child subpanels.
@willrennie  I would greatly appreciate if this is still considered as a feature and carried out in the repo.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Subpanel Save when visible in EditView does not hide form after save complete](https://trello.com/c/UpSxtUzP/61-subpanel-save-when-visible-in-editview-does-not-hide-form-after-save-complete)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
